.byebug_history

```diff
@@ -0,0 +1,48 @@
+c
```

Comment:
This file should be removed.

Reply:
Of course, sorry. Should i add it to gitignore?

Reply:
If you want to use byebug regularly in the future, it probably makes sense to add it (otherwise you need to think about it every time not to add it). You can add it in a separate commit here or create a new PR for it.

Reply:
Ok, i'll add a commit.
I'd also like to add awesome_print (https://github.com/awesome-print/awesome_print) if you don't mind.

Reply:
Thanks.

I'm not sure about awesome_print, I think we want to keep as little dependencies as possible and for me `pp` (pretty print) was always enough. But I think we can discuss that in the future if we want to add that.

Reply:
Didn't know pp. Sounds good :+1: